package.json

```diff
@@ -4,15 +4,20 @@
   "description": "Javascript library for DHIS2",
   "main": "d2.js",
   "scripts": {
-    "documentation": "./node_modules/.bin/jsdoc -c jsdoc.json --readme ./README.md -u ./tutorials",
+    "babel": "babel src --out-dir build --source-maps --ignore '/**/__tests__/*','/**/__mocks__/*','/**/__fixtures__/*'",
+    "build": "npm run clean && npm run babel && npm run build:dev && npm run build:prod",
```

Comment:
Why does this need to run the babel command? It already builds the dev and prod version. Do we have a third version? (That's what I meant by not knowing what the `babel` command does)

Reply:
So and what you're seeing in build is that after building the version that can be distributed over npm, it also builds standalone versions that can be included with a `<script src="">` directly with webpack.

Reply:
I talked to @varl about this, and we were thinking of just removing the webpack build entirely. That way this lib can just be used as a es6 or commonjs import. But we'll have to check first if anyone's including it directly on their page, seems unlikely, but still.

Reply:
I see, makes sense!